addons/compact-profile/addon.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Compact Scratch Profile Sliders",
+  "description": "Rearranges the profile sections into a cleaner two-column layout by pairing sliders side by side, making profiles more compact and readable.",
+  "credits": [
+    {
+      "name": "VIGARPAST_777",
+      "link": "https://scratch.mit.edu/users/VIGARPAST_777/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/users/*/"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyles.css",
+      "matches": ["https://scratch.mit.edu/users/*/"]
+    }
+  ],
+  "settings": [],
+  "versionAdded": "1.44.0",
+  "tags": ["profiles", "ui", "featured"]
```

The list of tags can be found [here](https://scratchaddons.com/docs/reference/addon-manifest/#tags) along with information on the rest of the `addon.json`.
```suggestion
  "tags": ["community", "profiles"]
```